index.html

```diff
@@ -3,9 +3,20 @@
   <head>
     <title>EST UI Widget</title>
     <link rel="stylesheet" type="text/css" href="style.css">
-    <script type="text/javascript" src="javascript/est-widget.js"></script>
+
+      <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
Is it better to ask users to include JQuery, or should our code do it? (Doesn't the widget code [already do this](https://github.com/GovWizely/est-ui-widget/pull/2/files#diff-4220180f0c119d766886877485f1806aR4)?)